go-controller/pkg/util/subnet_annotations.go

```diff
@@ -23,6 +23,10 @@ import (
 //       {
 //         "default": "100.64.2.0/29"
 //       }
+//     k8s.ovn.org/node-dnat-and-snat-ip: |
```

Comment:
lets not add annotation with this name. node-localnet-ips

Reply:
ok

Reply:
fix the comment to be the new annotation

Reply:
ok